test/Microsoft.Health.Functions.Examples/Sorting/DistributedSorter.cs

```diff
@@ -0,0 +1,93 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EnsureThat;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Health.Operations.Functions.DurableTask;
+
+namespace Microsoft.Health.Functions.Examples.Sorting;
+
+public class DistributedSorter
+{
+    private readonly SortingOptions _options;
+
+    public DistributedSorter(IOptions<SortingOptions> options)
+        => _options = EnsureArg.IsNotNull(options?.Value, nameof(options));
+
+    [FunctionName(nameof(InsertionSortAsync))]
+    public async Task<IReadOnlyList<int>> InsertionSortAsync([OrchestrationTrigger] IDurableOrchestrationContext context, ILogger logger)
```

Comment:
This is a pretty basic orchestration, but the hope is that it illustrates an durable function that calls multiple activities and even restarts via `ContinueAsNew`